src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/search.html.twig

```diff
@@ -41,6 +42,13 @@
                 </span>
             </span>
         </button>
+        <button type="reset" class="sf-button">
```

Comment:
The HTML of the buttons has changed. This:

```html
<button type="reset" class="sf-button">
    <span class="border-l">
        <span class="border-r">
            <span class="btn-bg">RESET</span>
        </span>
    </span>
</button>
```

should be replaced by:

```html
<button type="reset" class="btn">Reset</button>
```

Add the `btn-sm`, etc. classes if you need to change the button size.